include/revng/Support/IRHelperRegistry.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/IR/Module.h"
+
+#include "revng/Support/ManagedStaticRegistry.h"
+
+struct RegisteredIRHelper {
+public:
+  std::string Name;
+  std::string Description;
+
+public:
+  const std::string &key() const { return Name; }
+};
+using RegisterIRHelper = RegisterManagedStaticImpl<RegisteredIRHelper>;
```

Comment:
`IRHelperRegistry`?

Reply:
Actually, on a second thought, `RegisterIRHelper` is a better name. Mostly because that's the type of the object used for adding new helpers. It's weird when these are created like
```cpp
IRHelperRegistry NewPCHelper("newpc", "absent after remove-newpc-calls");
```

Don't you agree?

Reply:
Right, using an imperative name here is the right thing to do.